ricecooker/commands.py

```diff
@@ -157,18 +163,35 @@ def prompt_token(domain):
             config.LOGGER.error("Invalid token. Please login to {0}/settings/tokens to retrieve your authorization token.".format(domain))
             prompt_token(domain)
 
-def prompt_resume():
-    """ prompt_resume: Prompt user to resume last session if one exists
+def check_version_number():
+    response = config.SESSION.post(config.check_version_url(), data=json.dumps({"version": __version__}))
+    response.raise_for_status()
+    result = json.loads(response._content.decode('utf-8'))
+
+    if  result['status'] == 0:
+        config.LOGGER.info(result['message'])
+    elif result['status'] == 1:
+        config.LOGGER.warning(result['message'])
+    elif result['status'] == 2:
+        config.LOGGER.error(result['message'])
+        if not prompt_yes_or_no("Continue anyways?"):
+            sys.exit()
```

Comment:
My thought was this prompt should go on status==1, and for status==2, it shouldn't allow them to continue at all (always exit). This allows us to ban misbehaving versions completely, which could be important.